#Call analyse on file
string input_file
string output_file
---